packages/components/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@carto/airship-components",
-  "version": "0.0.0",
+  "version": "1.0.0-alpha.26",
```

Comment:
Reading lerna docs, it seems that we could skip individual version for the packages, as we would follow **fixed mode** (both packages always coordinated): https://www.npmjs.com/package/lerna#fixedlocked-mode-default Right?

Reply:
I think fixed mode makes all packages to have the same version

Reply:
Correct, didn't we want just that @jesusbotella ?

Reply:
yes :)

Reply:
Yep! 😉